README.md

```diff
@@ -465,6 +465,19 @@ Spline will pick it up automatically.
 <a id="building"></a>
 ### Building for different Scala and Spark versions
 
+**Note:** The project requires Java version 1.8 (strictly) and [Apache Maven](https://maven.apache.org/) for building.
```

Comment:
Why Java 11 does not work?

Reply:
Because of some test dependencies, I don't remember exactly at the moment, but it was discussed X times before. Agent works on any Java supported by Spark, but requires strictly 1.8 for building.